### PR TITLE
Feat : Added Drawer/List component

### PR DESCRIPTION
**Pull request by shlok30**
Please review my Drawer/List Component.

Site is not responsive as of now.

Link - https://lucid-kowalevski-e65406.netlify.app/components/drawer.html

For some reason the carbon.sh i-frame is not coloured this time.

Thanks

### CODE REVIEW

File: Components/drawer.html [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel = "stylesheet" href = "/main.css">
+
+    <link
+      rel="stylesheet"
+      href="https://fonts.googleapis.com/icon?family=Material+Icons"
+    />
+    <title>Navigation</title>
+</head>
```
Comment (mohitdhatrak) [inline Components/drawer.html]: update the title

**Review by mohitdhatrak — COMMENTED** [review]
Just the one correction, after that good to merge

File: Components/drawer.html [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+                <li><a href="navigation.html">Navigation</a></li>
+                <li><a href="slider.html">Slider</a></li>
+                <li><a href="drawer.html" class = "active-link">Drawer/List</a></li>
+            </ul>
+        </aside>
+
+        <div class = "flex flex-column main gap-m debug">
+            <h1>Drawer Component</h1>
+
+            <div class = "container light-border quarter-width" style = "max-width:100%">
```
Comment (viru2001) [inline Components/drawer.html]: you should avoid inline styling

**Review by viru2001 — COMMENTED** [review]
gave one suggestion. everything else is good

**Pull request merged**